subsys/net/ip/tcp.c

```diff
@@ -730,6 +730,8 @@ static int tcp_data_get(struct tcp *conn, struct net_pkt *pkt, size_t *len)
 
```

Comment:
Just noticed a typo in commit subject of `net: tcp: Implement recieive window handling`
s/recieive/receive/